application_helper/helper_functions.js

```diff
@@ -18,13 +18,20 @@ async function coordinates(address) {
 // might want to have models for data to format forecast
 async function fetchForecast(address) {
   let coords = await coordinates(address);
-  let response = await fetch(`https://api.darksky.net/forecast/${process.env.DARK_SKY_API}/${coords.lat},${coords.lng}`);
+  let response = await fetch(`https://api.darksky.net/forecast/${process.env.DARK_SKY_API}/${coords.lat},${coords.lng}?exclude=minutely,alerts,flags`);
   let forecast = await response.json();
   return forecast;
 };
 
+async function fetchForecastFav(address) {
```

Comment:
@scottzero what do you think of this?

Reply:
# Great job with the refactor. You were really able to condense these lines down and dry up your code. 
# Keep up the good work : )